--- a/src/od-pandora/pandora.cpp	2016-08-27 22:37:18.775421171 +0200
+++ b/src/od-pandora/pandora.cpp	2016-08-27 22:37:54.329147635 +0200
@@ -721,7 +721,6 @@
   
   if(currVSyncRate != hz)
   {
-    snprintf((char*)cmd, 64, "sudo /usr/pandora/scripts/op_lcdrate.sh %d", hz);
     system(cmd);
     currVSyncRate = hz;
     return true;
@@ -738,7 +737,6 @@
 
 	if(currprefs.pandora_cpu_speed != lastCpuSpeed)
 	{
-		snprintf((char*)speedCmd, 128, "unset DISPLAY; echo y | sudo -n /usr/pandora/scripts/op_cpuspeed.sh %d", currprefs.pandora_cpu_speed);
 		system(speedCmd);
 		lastCpuSpeed = currprefs.pandora_cpu_speed;
 		cpuSpeedChanged = true;
